### ISSUE TITLE
Does the current design support the export of the containers after Loading the load of the containers (LCL) ?

### ISSUE THREAD

**Issue by aalhadi** [NAFITH/IraqWeb#427]
@AdnanIsmail 
After applying the LCL scenario, the containers (physical containers) will be located in the berth, Exporting such  containers is a valid scenario to deal with 
Does the current design of the system support it?

**Comment by hhirzallah** [comment]
Yes you can move the empty container (physical)  from berth to vessel after LCL
